src/templates/common/contentFooter/index.tsx

```diff
@@ -25,7 +25,10 @@ export function ContentFooter({
   }
 
   return (
-    <div className={`last-updated ${wrapperClasses}`}>
+    <div
+      data-testid="content-footer"
```

Comment:
Just added these test IDs to check for the presence of the components, not a full integration. I'm largely relying on static type checking to make sure we're passing the correct data to the child components from `<HealthCareLocalFacility>`. (Which, of course, means if we're accidentally passing the _wrong_ value of the _correct_ type, it won't catch it, but.....I don't _think_ we're at risk of that with the components I added `data-testid` to. 🤞 Caveat: I didn't verify that, so I could be wrong. 😬 )